targets/lenovo-x1-carbon.nix

```diff
@@ -466,6 +474,58 @@
                     ];
                     borderColor = "#337aff";
                   }
+                  {
+                    name = "element";
+                    packages = [pkgs.element-desktop pkgs.gpsd pkgs.element-gps];
+                    macAddress = "02:00:00:03:08:01";
+                    ramMb = 4096;
```

Comment:
We are quite soon going to run out of RAM on the Lenovo laptop, if we have to reserve 4GB of RAM just for the sake of displaying map on the webview :D

Reply:
That was the proposed solution from weekly meeting.
Application memory consumption was profiled. The app won't even start within 1GB RAM.